src/modules/reading-room/services/visitor-space/vistor-space.service.ts

```diff
@@ -58,12 +63,13 @@ export class VistorSpaceService {
 		const formData = new FormData();
 
 		// Set form data
-		formData.append('color', values.color ?? '');
-		formData.append('image', values.image ?? '');
+		values.color && formData.append('color', values.color ?? '');
+		values.image && formData.append('image', values.image ?? '');
```

Comment:
Overbodige fallback?
```suggestion
		values.color && formData.append('color', values.color);
		values.image && formData.append('image', values.image);
```